pyproject.toml

```diff
@@ -13,6 +13,7 @@ dependencies = [
     "contourpy >=1",
     "numpy >=1.11.3",
     "packaging >=16.8",
+    "pandas >=1.2",
```

Comment:
Pulled this version out of a hat. Open to other suggestions, should we continue.